src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla57515.cs

```diff
@@ -1,25 +1,37 @@
 ﻿using NUnit.Framework;
+using NUnit.Framework.Legacy;
 using UITest.Appium;
 using UITest.Core;
 
 namespace Microsoft.Maui.TestCases.Tests.Issues;
 
 public class Bugzilla57515 : _IssuesUITest
 {
+	const string ZoomContainer = "zoomContainer";
+
 	public Bugzilla57515(TestDevice testDevice) : base(testDevice)
 	{
 	}
 
 	public override string Issue => "PinchGestureRecognizer not getting called on Android ";
 
-	// [Test]
-	// [Category(UITestCategories.Gestures)]
-	// [FailsOnIOS]
-	// public void Bugzilla57515Test()
-	// {
-	// 	RunningApp.WaitForElement(ZoomContainer);
-	// 	RunningApp.WaitForElement("1");
-	// 	RunningApp.PinchToZoomIn(ZoomContainer);
-	// 	RunningApp.WaitForNoElement("1"); // The scale should have changed during the zoom
-	// }
+	[Test]
+	[Category(UITestCategories.Gestures)]
+	[FailsOnIOS("It already failed in Xamarin.UITest on this platform.")]
+	[FailsOnMac("It already failed in Xamarin.UITest on this platform.")]
+	[FailsOnWindows("Cannot find the element ZoomContainer. Investigate the reason.")]
```

Comment:
Use if/def instead of FailsOnWindows so we don't lose track that this was working on Xamarin UITest

Reply:
Ok, changed, could you review it again?